src/lib/core/CHIPConfig.h

```diff
@@ -2394,3 +2394,35 @@ extern const char CHIP_NON_PRODUCTION_MARKER[];
 #define CHIP_COMMISSIONING_HINT_INDEX_PRESS_RESET_SECONDS_WITH_POWER 10
 #define CHIP_COMMISSIONING_HINT_INDEX_PRESS_RESET_UNTIL_BLINK_WITH_POWER 11
 #endif
+
+#ifndef CHIP_IM_MAX_NUM_COMMAND_HANDLER
```

Comment:
nit: can you document these settings now or sometime in the future?

Reply:
They should be included in this PR, updated, thanks.

Reply:
Thank you!